python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -871,33 +878,35 @@ def load(self, ids, load_backup=False):
 
     def _handle_load_exception(self, err, fn, this_id, load_backup):
         if isType(err, XMLFileError):
-             logger.error("XML File failed to load for Job id: %s" % str(this_id))
-             logger.error("Actual Error was:\n%s" % str(err))
+            logger.error("XML File failed to load for Job id: %s" % str(this_id))
+            logger.error("Actual Error was:\n%s" % str(err))
 
         if load_backup:
-             logger.debug("Could not load backup object #%i: %s", this_id, str(err))
-             raise InaccessibleObjectError(self, this_id, err)
+            logger.debug("Could not load backup object #%s: %s" % (str(this_id), str(err)))
+            raise InaccessibleObjectError(self, this_id, err)
 
-        logger.debug("Could not load object #%i: %s", this_id, str(err))
+        logger.debug("Could not load object #%s: %s" % (str(this_id), str(err)))
 
         # try loading backup
         try:
-             self.load([this_id], load_backup=True)
-             logger.warning("Object '%s' #%i loaded from backup file - the last changes may be lost.", self.registry.name, this_id)
-             return True
+            self.load([this_id], load_backup=True)
+            logger.warning("Object '%s' #%s loaded from backup file - the last changes may be lost." % (str(self.registry.name), str(this_id)))
```

Comment:
Absence of errors does not make it correct. It adds unnecessary noise to the code and makes it harder to read.